### PR TITLE
cp: add traditional Chinese translation

### PR DESCRIPTION
**Pull request by extend1994**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by kbdharun** [comment]
@extend1994 Welcome to tldr, Can you sign the CLA in the comment above?

**Comment by extend1994** [comment]
Hi @kbdharun, I have signed the CLA. Thanks for the welcome.

File: pages.zh_TW/linux/cp.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`cp -vr {{要被複製的資料夾路徑/要被複製的資料夾名稱}} {{目標路徑}}`
+
+- 透過 "互動模式" 複製文字檔到新位置 (當新位置有相同名稱的檔案時，需要先經過使用者的同意，方能複製到新位置)
+
+`cp -i {{*.txt}} {{目標路徑}}`
+
+- 複製檔案或資料夾 "捷徑" 指向的 "真實" 檔案或資料夾:
+
+`cp -L {{捷徑}} {{目標路徑}}`
```
Comment (blueskyson) [inline pages.zh_TW/linux/cp.md:28]:
```suggestion
- 複製檔案或「捷徑」指向的「真實」檔案或資料夾：

`cp -L {{捷徑}} {{目錄/完整/目標路徑}}`
```

File: pages.zh_TW/linux/cp.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- 使用完整的檔案路徑與檔案名稱，來創建跟原本目錄結構相同的資料夾與檔案
+
+`cp --parents {{要被複製的檔案路徑/要被複製的檔案檔名.檔案副檔名}} {{目標路徑/新檔名.檔案副檔名}}`
```
Comment (blueskyson) [inline pages.zh_TW/linux/cp.md:32]:
```suggestion
- 使用完整的檔案路徑與檔案名稱，來創建跟原本目錄結構相同的目錄與檔案：

`cp --parents {{檔案/完整/路徑}} {{檔案/完整/路徑}}`
```

**Review by blueskyson — COMMENTED** [review]
We have a translation template of `path/to/file` in [common-arguments.md](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md).

**Comment by extend1994** [comment]
Hi @blueskyson, I have modified the content as you suggested. Feel free to help me review it. Thanks!

**Comment by navarroaxel** [comment]
@extend1994 could you please delete the binary file?

File: pages.zh_TW/linux/cp.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# cp
+
+> 複製檔案與資料夾。
+> 更多資訊可參考：<https://www.gnu.org/software/coreutils/cp>.
+
+- 複製一個檔案到新位置，並重新命名：
+
+`cp {{檔案/完整/檔案.副檔名}} {{檔案/完整/新檔案.副檔名}}`
+
+- 複製一個檔案到目錄內，但保留檔案名稱：S
```
Comment (blueskyson) [inline pages.zh_TW/linux/cp.md:10]:
```suggestion
- 複製一個檔案到目錄內，但保留檔案名稱：
```

**Review by blueskyson — APPROVED** [review]
Please delete `.cp.md.swp`.

**Comment by extend1994** [comment]
@navarroaxel @blueskyson Thank you both. 👍 The unnecessary "S" and swap file had been deleted.
Have been working overtime these days, sorry for the late response.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**